### ISSUE TITLE
Waves Material Animation and Angular Animation don't work on built apk's

### ISSUE THREAD

**Issue by shaulhadar** [zachsoft/Ionic-Material#4]
Hello Again Zach, i did some tests and tried to compile the list page from your demos, it looks great and behaves great when viewing from chrome.  i than went to intel xdk and built the project in regular android, and also in crosswalk, and the application works great, but there are not animations working for the enter page, is there any additional step to make them work in the built app?

Thanks,
Shaul.

**Comment by zachsoft** [comment]
Makes sense - this will be fixed in accordance with the routing changes I'm working on rolling out with the next release. Hang tight and thanks for the feedback.

**Comment by shaulhadar** [comment]
Thanks Zach
